tests/python/unittest/test_operator.py

```diff
@@ -2413,6 +2413,47 @@ def test_l2_normalization():
                         check_l2_normalization((nbatch, nchannel, height, width), mode)
 
 
+def npy_layer_norm(data, gamma, beta, axis=1, eps=1E-5):
+    if axis < 0:
+        axis += data.ndim
+    broadcast_shape = [1 for _ in range(data.ndim)]
+    broadcast_shape[axis] = data.shape[axis]
+    mean = data.mean(axis=axis, keepdims=True)
+    var = data.var(axis=axis, keepdims=True)
+    std = np.sqrt(var + eps)
+    out = np.reshape(gamma, broadcast_shape) * (data - mean) / std + \
+          np.reshape(beta, broadcast_shape)
+    return out
+
+
+def check_layer_normalization(in_shape, axis, eps, dtype=np.float32):
+    ctx = default_context()
+    data = np.random.normal(0, 1, in_shape).astype(dtype)
+    gamma = np.random.normal(0, 1, (in_shape[axis],)).astype(dtype)
+    beta = np.random.normal(0, 1, (in_shape[axis],)).astype(dtype)
+    data_s = mx.symbol.Variable('data')
+    gamma_s = mx.symbol.Variable('gamma')
+    beta_s = mx.symbol.Variable('beta')
+    out_s = mx.symbol.LayerNorm(data=data_s, gamma=gamma_s, beta=beta_s, axis=axis, eps=eps)
+    exe = out_s.simple_bind(ctx, data=in_shape)
+    exe.arg_dict['data'][:] = data
+    exe.arg_dict['gamma'][:] = gamma
+    exe.arg_dict['beta'][:] = beta
+    out_nd = exe.forward()[0]
+    out = npy_layer_norm(data, gamma, beta, axis, eps)
+    assert_allclose(out, out_nd.asnumpy(), 1E-4, 1E-4)
```

Comment:
Is this the correctness test?

Reply:
Yes, it compares it with a numpy version.